src/libraries/System.Text.RegularExpressions/gen/Stubs.cs

```diff
@@ -38,6 +35,10 @@ internal static class StringExtensions
     {
         public static string Create<TState>(int length, TState state, SpanAction<char, TState> action)
         {
+            if (length == 0)
+            {
+                return string.Empty;
+            }
```

Comment:
Where is this being used with length 0 and where does that optimization matter?

Reply:
Reverted.